### ISSUE TITLE
[A11y] Work Show Page-Multiple - Link Purpose

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#2008]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j
https://staging.oregondigital.org/concern/documents/df727057n
https://staging.oregondigital.org/concern/images/1g05fb620

The most obvious concern here is that there are just way too many links. It isn't a fail, but I would love to see a way to get some of them "organized" somehow so they aren't always present.

Real problems:

- **Low**: Facebook and Twitter links should use an accessible name that reflect the page on which you land, e.g., instead of "Facebook", "Post to Facebook". It could be argued the surrounding context makes these links good enough, hence the "Low" here.
- **High**: Mentioned elsewhere, but all the "i" icon links have very broken accessible names, "Opens in new window". These need to describe the destination. Preferably these use the destination's title, if it's meaningful enough. If it's not meaningful, or automatically getting said title is too difficult, a more vague but meaningful description would suffice, such as "name authority record for X: opens in a new window" (please don't use this text without consulting with somebody smarter than me - it's just an example and probably sucks)
- **Low**: (Compound Parent primarily) Pagination links probably need extra context since there are multiple pagination "widgets" on the page. There are numerous ways this could be considered a success or a failure of WCAG (e.g., it passes because of the surrounding context; it fails because we end up with multiple links that have different destinations but the same accessible name), so this one's tricky to say how bad it is, if it's even a failure.
- **Medium**: "REVIEW AND APPROVAL" should ideally be a disclosure, which requires a "button" role and proper keyboard implementation of a button (it's a link and uses a simple link role and behavior).
- **High**: There's an invisible link to the homepage somewhere. I mentioned it in the universal viewer ticket (#1992), but I wasn't sure if it's ours or something built into UV.
- **Low**: As mentioned elsewhere, the metadata links aren't clear in function; this is usability but it will affect certain user groups more significantly than others. Some explanation of the metadata might be helpful, but the links themselves shouldn't get any extra hidden context, as that would just make for way too much verbosity when listing links.
- **High**: All images in "IS PART OF", "RELATED ITEMS", and "CONTAINED IN THIS OBJECT" lack accessible text. This is already mentioned in #1990, but bears repeating. The accessible name for these images should be their link destination.
- **High** All links to child objects and related items have the same accessible name, "Danforth Cooke & Co., Patterson, NJ. Engine purchases by OSN...". These need to have a unique name since they point to different pages. Ideally these works would start with the page or sequence number so it's immediately obvious which page a link goes to, and the links could continue to be abbreviated since the critical differentiation would remain.
- **High**: Image links under "WORKS AND FILES" all have the accessible name "file details". This should be dealt with in #2018 because merely changing the alt attribute would actually be less effective a fix.

### Suggested Fix

I tried to detail fixes above, so hopefully that's enough information. Generally follow the link guidelines Accessibility Insights gives with some thought to places where these suggestions may make things more confusing (e.g., destination page title isn't meaningful to convey the link's purpose):

- If a link navigates to a document or web page, the name of the document or page is sufficient.
- Links with different destinations should have different link text.
- Links with the same destination should have the same link text.

### Can anybody QA or does this need an expert?

Needs expert

### QA Steps

A lot of links here - probably a lot faster to have an expert test. If it's not an option, testing means using a tool, not just scanning HTML by hand, and verifying every single link has an accessible name that matches its purpose.

**Comment by jechols** [comment]
I'm passing this one, but there are issues still and I want to make sure it's on record that I object 😝 

- Pagination is still ambiguous but this isn't a hard failure, sadly
- Share links still kinda suck, but again, not a hard failure
- The invisible link still exists. This is a hard failure, but it's mentioned in another ticket.
- The links with the same accessible name are still a problem. I'll create a new ticket for this since it will be easier to tackle a single-item ticket.

Also worth noting, image links under "WORKS AND FILES" *are good* even though they all have the accessible name "file details". This is **not** a failure anymore. It will seem like it on first glance, but these are in fact differentiated by the table row header that was added as part of another ticket.
